### ISSUE TITLE
App not loading

### ISSUE THREAD

**Issue by oaksprout** [valory-xyz/olas-operate-app#203]
This has been reported by one user – waiting for more logs/information.

1. <img width="767" alt="image" src="https://github.com/valory-xyz/olas-operate-app/assets/66292936/b0b3fe1f-a5eb-4353-903c-d3b624bfd475">
2. <img width="318" alt="image" src="https://github.com/valory-xyz/olas-operate-app/assets/66292936/fe175fd3-6daa-42cc-8aef-93f719014e53">

Contents of operate.log (looks fine):
```
Checking brew installation
Checking python installation
Creating required directories
Checking tendermint installation
Checking if upgrade is required
Installing pearl CLI
```

**Comment by oaksprout** [comment]
Something I noticed about this issue – the people who are afflicted mostly are people who were previously able to run the app, or at least who tried to run it early on. cc @0xArdi

**Comment by oaksprout** [comment]
1 afflicted user's setup:

Version 0.1.0-rc71 (0.1.0-rc71)
Using an M2 Macbook Air running Sonoma 14.5 (23F79)

cc @0xArdi

**Comment by oaksprout** [comment]
Further info from another user:

![image](https://github.com/user-attachments/assets/627accb6-6a19-4a06-96c7-9fdeb64b8eab)

**Comment by oaksprout** [comment]
It seems like I’ve replicated the issue.

Here’s what I did:
1. I had Pearl open – rc80
2. I renamed my .operate to .operate1 (I cycle between a few agents locally)
3. This seemed to break the app a bit – I noticed it spawned another .operate
4. I tried deleting this multiple times but it kept getting recreated. Found that there were some dangling aea and tendermint processes so I manually killed them and it was fixed.
5. I resuscitated an old .operate folder (it was called something like .operate_broken, can’t remember what was wrong with it)
6. Restarted the app and I got the loading screen of death
